lithops/standalone/standalone.py

```diff
@@ -146,15 +146,26 @@ def invoke(self, job_payload):
         workers = job_payload['workers']
 
         total_workers = min(workers, total_calls // chunksize + (total_calls % chunksize > 0)
-                            if self.exec_mode == 'create' else 1)
+                            if self.exec_mode in ['create', 'reuse'] else 1)
 
         def start_master_instance(wait=True):
             if not self._is_master_service_ready():
                 self.backend.master.create(check_if_exists=True, start=True)
                 if wait:
                     self._wait_master_service_ready()
 
-        if self.exec_mode == 'create':
+        def get_workers_on_master():
+            workers_on_master = []
+            try:
+                cmd = (f'curl -X GET http://127.0.0.1:{STANDALONE_SERVICE_PORT}/workers -H \'Content-Type: application/json\'')
+                workers_on_master = json.loads(self.backend.master.get_ssh_client().run_remote_command(cmd))
+                self.backend.master.del_ssh_client()
```

Comment:
Create a ssh client has an overhead of 2-3 seconds (depending of the client and the server locations). `get_workers_on_master()` is called multiple times in a row in the `run()` method. So delete the ssh client here will crate an overhead of 4-5 seconds. The ssh client is already deleted at the end of the `run()` method, so I think it is not needed to delete it here.

Reply:
Done